lib/client.js

```diff
@@ -223,7 +224,7 @@ class Client extends DispatcherBase {
     this[kConnector] = connect
     this[kSocket] = null
     this[kPipelining] = pipelining != null ? pipelining : 1
-    this[kMaxHeadersSize] = maxHeaderSize || 16384
+    this[kMaxHeadersSize] = maxHeaderSize || http.maxHeaderSize || 16384
```

Comment:
`http.maxHeaderSize` has a default value of `16KiB`, meaning that the property will always have a value.

Would it be better to just default to `http.maxHeaderSize` if we are going to be depending on this variable? Anyways is the same default value we have

cc: @ronag @mcollina

Reply:
That makes sense since that value is also read-only. I removed the default 16384. Let me know if it looks good. :+1: